CoroutineTemplate/app/src/main/java/co/nimblehq/coroutine/ui/screens/compose/composables/UserItem.kt

```diff
@@ -40,11 +40,11 @@ fun UserItem(
             ) {
                 with(user) {
                     Text(
-                        text = name,
+                        text = name.orEmpty(),
```

Comment:
perhaps for later, we should create a UserUiModel that handles this logic from the UserEntity. As `name` can be null, then `UiModel` can prepare the `empty string` for this field instead of asking the View to manage the null case itself

Reply:
@sleepylee I think I need to return the `UserModel` instead of `UserEntity`